features/patch-after-fetch-svn.feature

```diff
@@ -0,0 +1,80 @@
+Feature: Patch after fetching from git repo
+
+    Sometimes a patch needs to be applied after fetching. *DFetch* makes it
+    possible to specify a patch file.
+
+    @wip
```

Comment:
There are some stray `@wip` tags in this file 😉 , probably a development artifact

Reply:
Removed